src/index.rs

```diff
@@ -1133,4 +1132,211 @@ mod tests {
         assert!(new2 > old2);
         assert!(new2 > new1);
     }
+
+    const FILE_DIR_1: &str = "folder_1";
+    const FILE_DIR_2: &str = "folder_2";
+    const FILE_NAME: &str = "test_";
+    const FILE_COUNT: i32 = 10;
+
+    fn generate_random_update(
+        root_path: PathBuf,
+        folder_name: Option<&str>,
+        name: Option<&str>,
+    ) -> i32 {
+        let mut rng = rand::thread_rng();
+        let mut rnd_num = rng.gen_range(1..=4);
+
+        let mut folder_1 = root_path.clone();
+        let mut folder_2 = root_path.clone();
+        folder_1.push(FILE_DIR_1);
+        folder_2.push(FILE_DIR_2);
```

Comment:
Do we really need 2 folders?

Reply:
> Do we really need 2 folders?

It is necessary for file movement function.

Reply:
We can also just move to new random filename. This way we can apply move operation multiple times, no need to track in which of the two the file resides.